q2_demux/_summarize/assets/src/box.js

```diff
@@ -47,6 +57,21 @@ export default function plotBoxes(svg, data, x, y) {
         .selectAll('td')
           .data(d => d)
           .text(d => d);
+
+      let seqLenNote = `The minimum sequence length identified during subsampling was ${min_seq_len}`;
+      if (inTheDangerZone) {
+        seqLenNote = `This position (${position}) is greater than the minimum sequence length observed
+                      during subsampling (${min_seq_len} bases). As a result this plot is not based on
```

Comment:
Can we say `As a result, the plot at this position is not based on...` instead? @jairideout & @gregcaporaso, thoughts? My concern is the distinction between the boxplot (at a particular position), versus the overall plot.